deployment/chart/values.yaml

```diff
@@ -0,0 +1,12 @@
+# Default values.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+spec:
+  template:
+    metadata:
+    spec:
+      initContainers:
+      - image: acrmicroserviceswestus.azurecr.io/hexadite:latest 
+        env:
+        - value: https://microservices-deploy-kv.vault.azure.net 
```

Comment:
In the deployment yaml template, you are using ```.Values.HexaditeImage``` and ```.Values.DeploymentSecretsKeyVaultUrl``` but I dont seem there here. Also, do we need all of this other stuff?